src/StatusQ/Controls/StatusInput.qml

```diff
@@ -276,8 +276,12 @@ Item {
                             valid: result
                         }
                     }
-                    result.errorMessage = validator.errorMessage
                     errors[validator.name] = result
+
+                    // the only way to trigger bindings for var property
+                    errors = errors
```

Comment:
Tricky but smart :+1:

Reply:
I am actually surprised this works... :D

Reply:
It is written in documentation about this https://doc.qt.io/qt-5/qml-var.html#change-notification-semantics